homeassistant/components/sensor/postnl.py

```diff
@@ -107,3 +115,50 @@ def update(self):
         }
 
         self._state = len(status_counts)
+
+
+class PostNLletter(Entity):
```

Comment:
Is there any information on https://developers.home-assistant.io ore any other site that i can follow because i cant get it working. 

I am getting this error in HA. It is a useless error for me so i am kind off stranded here.
```
SyntaxError: invalid syntax
Fatal error while loading config: invalid syntax (postnl.py, line 73)
Failed config
  General Errors: 
    - invalid syntax (postnl.py, line 73)
Successful config (partial)
```

This is what i tried to do. 
[postnl.txt](https://github.com/home-assistant/home-assistant/files/2179977/postnl.txt)
[postnl.zip](https://github.com/home-assistant/home-assistant/files/2179996/postnl.zip) (includes postnl.py)

Reply:
Your configuration is not passing the validation.

https://developers.home-assistant.io/docs/en/development_validation.html

Reply:
Thanks. I really appreciate your help. 👏